### PR TITLE
Interactives UX: Modals

### PR DESCRIPTION
**Pull request by jmif**
- Adds simple base modal 
- Changes InteractiveModal to use BaseModal under the hood
- Add modal i18n
- Add final modal styling

### CODE REVIEW

File: ozaria/site/components/interactive/PageInteractive/common/ModalInteractive.vue [hunk 1]
```diff
@@ -1,90 +1,132 @@
...
     </div>
-  </transition>
+  </base-modal>
 </template>
 
-<script>
-  export default {}
-</script>
+<style lang="scss">
+  @import url('https://fonts.googleapis.com/css?family=Arvo&display=swap');
```
Comment (jmif) [inline ozaria/site/components/interactive/PageInteractive/common/ModalInteractive.vue]: TODO load font properly

File: ozaria/site/components/common/BaseModal.vue [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  <transition name="modal">
+    <div class="modal-mask">
+      <div class="modal-container">
+        <slot />
+      </div>
+    </div>
+  </transition>
+</template>
+
+<script>
```
Comment (AndrewJakubowicz) [inline ozaria/site/components/common/BaseModal.vue]: Is this script required?

File: ozaria/site/components/common/BaseModal.vue [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        opacity: 0;
+    }
+
+    .modal-leave-active {
+        opacity: 0;
+    }
+
+    .modal-enter .modal-container,
+    .modal-leave-active .modal-container {
+        -webkit-transform: scale(1.1);
```
Comment (AndrewJakubowicz) [inline ozaria/site/components/common/BaseModal.vue]: This line made me wonder how much we need to be prefixing properties.
I haven't prefixed my `transform` property when using it.

**Comment by jmif** [comment]
@AndrewJakubowicz will follow up with you re error

File: ozaria/site/components/common/BaseModal.vue (hunk 3 above)
Comment (jmif) [reply ozaria/site/components/common/BaseModal.vue]: Great question.  Vue uses an auto prefixer by default so this isn't necessary, will remove.

https://cli.vuejs.org/guide/css.html#postcss

File: ozaria/site/components/common/BaseModal.vue (hunk 2 above)
Comment (jmif) [reply ozaria/site/components/common/BaseModal.vue]: Seems like its not, removing.

**Comment by AndrewJakubowicz** [comment]
@jmif The only blocking change I can see is the font loading in china. Otherwise looks good to me.

Going to test resizing the cinematic off this PR now.

**Pull request merged**